### ISSUE TITLE
Change to the 'mode' arg in Plotly? 

### ISSUE THREAD

**Issue by lewi0332** [learn-co-students/applying-gradient-descent-lab-data-science-intro-000#2]
Hi,
Sorry if this isn't the correct path or best practice. I'm a new student working through the data science intro course. Along the way I see that a few functions have an error in the graph.py file. In every instance that the 'mode' argument is listed as 'line', Plotly returns an error.  Looks like the argument for a line graph is 'lines'.  I've mostly solved this on learn.co by copying in the functions directly into the notebook rather than getting them from the library, but probably easier for future learners if that could be fixed? Thanks and see you on campus soon! Then I'll learn the correct etiquette for raising an issue :)

**Comment by L-Lewis** [comment]
Agreed - could this be fixed so that it isn't necessary to manually copy over the graphing code in each lesson please?
